### ISSUE TITLE
framework not found Metal for architecture armv7

### ISSUE THREAD

**Issue by livingbeams** [vedi/share-bunch-unity3d#2]
I get this linker error trying to build a project in Xcode 5.1.1

I've tryed the indications in:
 http://www.appsee.com/blog/dev/xcode-new-link-frameworks-automatically-fix-framework-not-found-metal-for-architecture-armv7

Disabling the option "Link Framworks Automatically" doesn't remove the linker error.

I've also tryied to enable the "Enable Modules (C and Objective-C)" option, as indicated in another article, but it neither works.

**Comment by vedi** [comment]
if you remove `-ObjC` flag, does linker error disappear?

**Comment by livingbeams** [comment]
Yes, removing the -ObjC flag makes the error to disappear, but then if the SahreBunch.cs script is loaded in the project a runntime error is generated at the call: 
UIApplicationMain(arg .... 
in main.mm

libc++abi.dylib: terminating with uncauht exception of type NSException

**Comment by vedi** [comment]
ok, let's try the following:
1. go to `Other Linker Flags`
2. remove `-ObjC`
3. add `-force_load "$(SRCROOT)/Libraries/libShareBunch.a"`

We just apply the same rules, but for only library, not globally.

I hope it will help.

**Comment by livingbeams** [comment]
Thanks for your answer.

With the flag -force_load "$(SRCROOT)/Libraries/libShareBunch.a"
I get the same linker error again.

**Comment by vedi** [comment]
@livingbeams, have you managed to solve the issue?

**Comment by livingbeams** [comment]
No, I couldn't find out how to solve it
